src/test/java/org/xstefank/check/TitleCheckTest.java

```diff
@@ -3,22 +3,48 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.xstefank.TestUtils;
-
+import org.xstefank.model.yaml.RegexDefinition;
 import java.util.regex.Pattern;
 
 public class TitleCheckTest {
 
+    private RegexDefinition testTitle = new RegexDefinition();
```

Comment:
also please rename - just to clearly see that it's not directly title but more title validation definition